src/java/fr/paris/lutece/plugins/forms/export/AbstractFileGenerator.java

```diff
@@ -71,6 +88,8 @@ public abstract class AbstractFileGenerator implements IFileGenerator
 
     private List<FormExportConfig> _configList = null;
     private Form _form;
+    
+    protected static IFormResponseTemplateService _formResponseTemplateService = SpringContextService.getBean( "workflow-formspdf.formResponseTemplateService" );
```

Comment:
This service should belong to the  PDF generator classes.
The FormResponseTemplateService itself should be moved from the workflow-formspdf module to the Forms Service, as a generic service, to avoid unnecessary dependency.